index.html

```diff
@@ -4,16 +4,110 @@
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Exercitationem animi alias magnam provident sint omnis, accusantium cupiditate ducimus dolorum! Molestiae rem illum, similique labore obcaecati qui distinctio vel perferendis tenetur?</p>
+    </div>
+  </div>
+  <div class="skills page">
+    <img src="" /> 
+    <h3>sk1llz</h3>
+    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Animi vero, inventore nihil repellendus consequuntur dolor necessitatibus nesciunt aut tempora reiciendis aliquid ad maxime iste qui sed soluta voluptatem, velit harum!</p>
+    <div  class="text">
+      <h3></h3>
+        <p>
+          
+        </p>
+    </div>
+  </div>
+  <div class=" works page ">
+    <h3>w3rkz</h3>
+    <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Numquam, error dicta libero, nesciunt molestiae pariatur aspernatur in rem, nemo eveniet maxime nostrum doloremque maiores fuga. Odit iure quaerat voluptatibus molestiae?</p>
+  </div>
+  <div class=" resume page ">
+      <h3>r3sum3</h3>
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Soluta minus odit explicabo consequuntur numquam, obcaecati aperiam eum officiis recusandae. Tempore minus sit minima in. Nobis soluta aliquid doloribus necessitatibus exercitationem.</p>
+  </div>
+  <div class=" contact page">
+    <h3>Contact</h3>
+    <form action="/my-handling-form-page" method="post">
+      <div>
+        <label for="name">Name:</label>
+          <input type="text" id="name" name="user_name">
+      </div>
+
+      <div>
+        <label for="mail">E-mail:</label>
+        <input type="email" id="mail" name="user_email">
+      </div>
+
+      <div>
+        <label for="msg">Message:</label>
+        <textarea id="msg" name="user_message"></textarea>
+      </div>
+     
+      <div class="button">
+        <button type="submit">Send your message</button>
+      </div>
+    </form> 
+  </div> 
+  
+</section>
+</div>
+
+<!--<div class="scroll"><h1>z</h1></div>-->
+<!--<div class="scroll"><h1></h1></div>-->
+<!--<div class="scroll"><h1></h1></div>-->
+
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+<script type="text/javascript" src="js/jquery.min.js"></script>
+<script type="text/javascript" src="js/jquery.jInvertScroll.js"></script>
+<script type="text/javascript">
+(function($) {
+  $.jInvertScroll(['.scroll'],        // an array containing the selector(s) for the elements you want to animate
+      {
+      height: 6000,                   // optional: define the height the user can scroll, otherwise the overall length will be taken as scrollable height
+      onScroll: function(percent) {   //optional: callback function that will be called when the user scrolls down, useful for animating other things on the page
+          console.log(percent);
```

Comment:
you need to comment this out, as it is now it's spitting a number to the console every micro-scroll